File: Example Code/01 Variables, Data Types, Input and Output/Math_PerimeterCalculator.cpp
#include <iostream>
using namespace std;

int main()
{
    float width, length;
    cout << "PERIMETER CALCULATOR" << endl;
    
    cout << "Enter width: ";
    cin >> width;
    
    cout << "Enter length: ";
    cin >> length;
    
    float perimeter = 2 * width + 2 * length;
    
    cout << "Perimeter: " << perimeter << endl;    
    
    return 0;
}

File: Example Code/01 Variables, Data Types, Input and Output/Math_PerimeterCalculator.java
import java.util.Scanner;

public class Math_PerimeterCalculator
{
    public static void main( String args[] )
    {
        Scanner scanner = new Scanner( System.in );
        double width, length;
        
        System.out.println( "PERIMETER CALCULATOR" );
        System.out.print( "Enter the width and length: " );
        width = scanner.nextDouble();
        length = scanner.nextDouble();
        
        // Order of operations
        double area = 2 * width + 2 * length;
        
        System.out.println( "Area: " + area );
    }
}
